### DISCUSSION TITLE
Select Skyculture

### DISCUSSION THREAD

**Discussion by anisio82** [Stellarium/stellarium#3064] [Q&A] [answered]
Hello everybody

I work on a science museum in Curitiba, a city of Brazil. We are creating a new interactive experiment on ethnoastronomy and archaeoastronomy for our exhibition. One of the ideias is to use Stellarium in a large touchscreen, with the Sky Cultures feature, showing different constellations of the other civilizations. But, we are studing a way for the visitor select or change the culture without access the configuration menu. In summary, is there any way to create a menu on the screen (by script or otherwise), where the visitor could change the sky culture in an easy and direct way?

Thanks

**Comment by gzotti** [comment, marked as answer]
The RemoteControl plugin provides a web-based interface. You can customize your own interface that only allows changing skycultures. See the User Guide for details.

**Reply by anisio82** [reply]
Hello gzotti

Thanks for indication. I'll try
